manifests/0000_70_cluster-network-operator_03_deployment.yaml

```diff
@@ -53,6 +53,8 @@ spec:
           value: "5000"
         - name: OVN_CONTROLLER_INACTIVITY_PROBE
           value: "30000"
+        - name: EGRESS_ROUTER_IMAGE
+          value: "quay.io/dmellado/egress-router-test:latest"
```

Comment:
I realize this is probably why it's a WIP, but, yeah, make sure you update to the actual image when taking it out of WIP.